sktime/datatypes/_adapter/gluonts.py

```diff
@@ -11,8 +11,12 @@ def convert_pandas_to_listDataset(pd_dataframe: pd.DataFrame):
 
     Returns
     -------
-    gluonts.dataset.common.ListDataset
-        A gluonTS ListDataset formed from `pd_dataframe`
+    list_dataset: list
+        A list of dict where each dict represents a time series with the following keys:
+        - "start": The starting timestamp.
```

Comment:
there needs to be a newline before the first bullet point for the list to be rendered correctly in the docs